### PR TITLE
Add html and htmlAttribute escape hatches

### PR DESCRIPTION
**Pull request by anmolitor**

### CODE REVIEW

File: codegen/helpers/Html/WithContext/Internal.elm [hunk 1]
```diff
@@ -30,3 +30,11 @@ withContext f =
...
 withContextAttribute f =
     Attribute (\context -> runAttribute context <| f context)
+
+
+html : Html.Html msg -> Html context msg
+html = always >> Html
+
+
+htmlAttribute : Html.Attribute msg -> Attribute context msg
+htmlAttribute = always >> Attribute
```
Comment (miniBill) [inline codegen/helpers/Html/WithContext/Internal.elm]: Please write these down explicitly. `>>` is bad for performance

**Review by miniBill — CHANGES_REQUESTED** [review]
The change makes sense in general, but please avoid `always` and `>>`

File: codegen/helpers/Html/WithContext/Internal.elm (hunk 1 above)
Comment (anmolitor) [reply codegen/helpers/Html/WithContext/Internal.elm]: I didn't know! Is it bad because JS is not optimized for currying?

Comment (anmolitor) [reply codegen/helpers/Html/WithContext/Internal.elm]: I changed the implementation to not use any helper functions

**Comment by miniBill** [comment]
This closes #2

**Pull request merged**